acronym/acronym.py

```diff
@@ -1,2 +1,4 @@
+import re
+
 def abbreviate(words):
-    pass
+    return re.sub('[^A-Z]', '', words.replace("'", "").title())
```

Comment:
@zarniak That solution is ace! The more built-in tools, that do most of the job for you, you use - the better IMHO. Python comes with "batteries included" and it's one of its strengths.

The only thing I'd change is that I'd make 2 lines out of it. Otherwise, what reading this feels like:
- Ok, I'm going to use regex substitution to remove all non-capital letters in ... [STASH THAT THOUGHT]
- ... this string that is a result of couple operations on another string [COMPUTE, BACK TO OTHER THOUGH]

So something like this _feels_ more readable. I'd call that "nested" (in 1 line) vs "linear" (many lines) reading/understanding.
```
words_titled = words.replace("'", "").title()
return re.sub('[^A-Z]', '', words_titled)
```